src/main/java/org/sqlite/SQLiteJDBCLoader.java

```diff
@@ -62,9 +62,46 @@
      * @return True if SQLite native library is successfully loaded; false otherwise.
      */
     public static boolean initialize() throws Exception {
-        loadSQLiteNativeLibrary();
-        return extracted;
+        // only cleanup before first extract
+		if(!extracted) {
+			cleanup();
+		}
+		loadSQLiteNativeLibrary();
+		return extracted;
     }
+	
+	
+	/**
+	* Deleted old nativ libraries e.g. on Windows this are not removed on VM-Exit (bug #80)
+	*/
+	static void cleanup(){
+
+		final String ver = org.sqlite.SQLiteJDBCLoader.getVersion();
+		String tempFolder = new	java.io.File(System.getProperty("java.io.tmpdir")).getAbsolutePath();					
+		java.io.File dir = new java.io.File(tempFolder);
+		java.io.File [] files = dir.listFiles(new java.io.FilenameFilter() {
+			@Override
+			public boolean accept(java.io.File dir, String name) {
+				return name.startsWith("sqlite-" + ver);
```

Comment:
Use private variable e.g., `private final String prefix ="sqlite-" + ver` to minimize String generation cost.